test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -3652,10 +3652,18 @@ public async Task DurableEntity_ListEntitiesAsync_FetchInput(bool fetchInput, st
                 EntityName = "aaab",
                 LastOperationFrom = yesterday,
                 LastOperationTo = tomorrow,
-                FetchInput = fetchInput,
+                FetchState = fetchInput,

```

Comment:
Let's also rename this local variable to `fetchState`. I suggest we also update the name of the test method to be `DurableEntity_ListEntitiesAsync_FetchState`.